pom.xml

```diff
@@ -55,82 +53,91 @@
   </developers>
 
   <scm>
-    <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}.git</connection>
-    <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}.git</developerConnection>
-    <url>http://github.com/jenkinsci/${project.artifactId}</url>
-    <tag>HEAD</tag>
+    <connection>scm:git:git://github.com/${gitHubRepo}.git</connection>
+    <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
+    <url>https://github.com/${gitHubRepo}</url>
+    <tag>${scmTag}</tag>
   </scm>
 
-  <!-- Bring some sanity to version numbering... -->
   <properties>
-    <jenkins.version>2.222.4</jenkins.version>
-    <hpi.compatibleSinceVersion>1.5.0</hpi.compatibleSinceVersion>
-    <google.api.version>1.25.0</google.api.version>
-    <google.http.version>1.21.0</google.http.version>
-    <google.guava.version>30.0-jre</google.guava.version>
-    <google-oauth.version>1.0.0</google-oauth.version>
-    <storage.revision>158</storage.revision>
+    <changelist>999999-SNAPSHOT</changelist>
+    <gitHubRepo>jenkinsci/google-storage-plugin</gitHubRepo>
     <java.level>8</java.level>
-    <concurrency>5</concurrency>
-    <skipIts>true</skipIts>
+    <jenkins.version>2.319.1</jenkins.version>
+    <hpi.compatibleSinceVersion>1.5.0</hpi.compatibleSinceVersion>
+    <google-api.version>1.33.2</google-api.version>
+    <google-http.version>1.41.4</google-http.version>
+    <google-oauth.version>1.33.1</google-oauth.version>
+    <google-storage.revision>v1-rev20211201-1.32.1</google-storage.revision>
+    <guava.version>31.0-jre</guava.version>
```

Comment:
maybe we should have a PR with only this guava upgrade?

Reply:
done in #188